source/features/view-raw-file.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const createRawUrl = (href: string) => href.replace(/blob\//, 'raw/');
```

Comment:
This fails on user/repo containing blob. Use something like:

```js 
url = link.pathname.split('/')
url[3] = 'raw'; // Replaces 'blob'
return url.join('/');

Reply:
🤦‍♂️ Of course... I was worried about the path or the filename and forgot about the user/repo!